.mint/main.yaml

```diff
@@ -0,0 +1,39 @@
+on:
+  github:
+    push:
+      init:
+        commit-sha: ${{ event.git.sha }}
+        branch: ${{ event.git.branch }}
+    pull_request:
+      init:
+        commit-sha: ${{ event.git.sha }}
+        branch: ${{ event.git.branch }}
```

Comment:
I'm a little confused -- why is this `main.yml` but running on push to all branches and pull requests? We usually reserve `main.yml` for builds that run only on main. Seems like we could just call this `ci.yml` if we want it to run on all pushes and PRs?

Reply:
ah, I wasn't aware of this nomenclature. It was initially called `continious_integration.yaml`, but I noticed that the "main" file was called `main.yaml` in mint. I didn't make the connection to the branch name 🙂.

I don't mind changing it to something else

Reply:
cool, let's change it to `ci.yml` then